### PR TITLE
Fix 2 typos

### PR DESCRIPTION
**Pull request by dabcoder**
Minor corrections to the FAQ.

### CODE REVIEW

**Comment by brian-brazil** [comment]
Thanks!

File: content/docs/introduction/faq.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ sort_rank: 5
 Prometheus is an open-source systems monitoring and alerting toolkit
 with an active ecosystem. See the [overview](/docs/introduction/overview/).
 
-### How does Prometheus compare against \[other monitoring system\]?
+### How does Prometheus compare against \[other monitoring systems\]?
```
Comment (juliusv) [inline content/docs/introduction/faq.md]: This wasn't actually a typo, but a placeholder (thus the square brackets). But I guess the way it's now is ok too.

Comment (grobie) [inline content/docs/introduction/faq.md]: It'd actually make more sense without the brackets now. It'd also be more readable imho.

Comment (juliusv) [inline content/docs/introduction/faq.md]: Pushed to master.

**Pull request merged**